clients/go/admin/deviceflow/token_orchestrator.go

```diff
@@ -150,7 +154,7 @@ func (t TokenOrchestrator) FetchTokenFromAuthFlow(ctx context.Context) (*oauth2.
 	if len(t.ClientConfig.Scopes) > 0 {
 		scopes = strings.Join(t.ClientConfig.Scopes, " ")
 	}
-	daReq := DeviceAuthorizationRequest{ClientID: t.ClientConfig.ClientID, Scope: scopes}
+	daReq := DeviceAuthorizationRequest{ClientID: t.ClientConfig.ClientID, Scope: scopes, Audience: t.ClientConfig.Audience}
```

Comment:
Shouldn't this attempt to get the audience from the GetPublicClientInfo endpoint first?

Reply:
Its coming from GetPublicClientInfo object.
https://github.com/flyteorg/flyteidl/pull/314/files#diff-e9c73c8c26d2564bad2d6d4543f229c55470bd18b8cbb900832ec4a7b2e1ad70R20-R42

Reply:
cc : @EngHabu